### PR TITLE
Cleanup ban

### PR DESCRIPTION
**Pull request by wrrrzr**
Make BanManager more const correctly
Delete unused includes

### CODE REVIEW

File: src/server/ban.cpp [hunk 1]
```diff
@@ -7,11 +7,11 @@
 #include <fstream>
 #include "threading/mutex_auto_lock.h"
 #include <sstream>
-#include <set>
 #include "util/strfnd.h"
 #include "util/string.h"
 #include "log.h"
 #include "filesys.h"
+#include "exceptions.h"
```
Comment (appgurueu) [inline src/server/ban.cpp:14]: Why does this add an include?

Comment (sfan5) [reply src/server/ban.cpp:14]: it was previously in ban.h

**Review by sfan5 — APPROVED** [review]
lgtm

File: src/server/ban.cpp (hunk 1 above)
Comment (appgurueu) [reply src/server/ban.cpp:14]: nvm

**Review by JosiahWI — APPROVED** [review]

**Pull request merged**